config/locales/en/loans.en.yml

```diff
@@ -12,6 +12,8 @@ en:
     confirm_response_deletion: "WARNING: Are you sure you want to permanently delete these responses?"
     criteria: "Criteria"
     criteria_not_entered: "Criteria have not been entered yet."
+    default_name_message: "If left blank, a name will be automatically generated for this loan."
+    default_loan_name: "%{org} - %{date}"
```

Comment:
I'm not quite sure I follow you here. A string from I18n will only be html_safe if the key ends in `html`: http://guides.rubyonrails.org/i18n.html#using-safe-html-translations. But that shouldn't matter because you've removed the only html. 

So just changing `&ndash;` to - should be enough to remove the need for .html_safe, and I'm fine with that. But this doesn't need to be in here as there is nothing to translate.